### ISSUE TITLE
cargo tomlfmt

### ISSUE THREAD

**Issue by thomaseizinger** [actions-rs/meta#20]
We use the `cargo-tomlfmt` binary to enforce a certain formatting of the `Cargo.toml` files.

Specifically, we sort the list of dependencies alphabetically which helps with avoiding merge conflicts.

Would you consider creating an action that allows to use this tool without having to install it every time?
Installing it takes somewhere around 4 minutes.

References:
- https://github.com/tbrand/cargo-tomlfmt

**Comment by svartalf** [comment]
Yeah, `cargo install` command is incredibly slow, but at this point this thing is blocked by https://github.com/actions-rs/meta/issues/21.
I presume, it would be possible to make some kind of an opaque cache, which is automatically caches the artefacts generated by any `cargo install` command invoked; see linked issue for more details about that.
